gammapy/astro/darkmatter/spectra.py

```diff
@@ -237,4 +237,106 @@ def from_dict(cls, data):
         return cls(scale=scale, **data)
 
 
+class DarkMatterDecaySpectralModel(SpectralModel):
+    r"""Dark matter decay spectral model.
+
+    The gamma-ray flux is computed as follows:
+
+    .. math::
+        \frac{\mathrm d \phi}{\mathrm d E} =
+        \frac{\Gamma}{4\pi m_{\mathrm{DM}}}
+        \frac{\mathrm d N}{\mathrm dE} \times J(\Delta\Omega)
+
+    Parameters
+    ----------
+    mass : `~astropy.units.Quantity`
+        Dark matter mass
+    channel : str
+        Annihilation channel for `~gammapy.astro.darkmatter.PrimaryFlux`
+    scale : float
+        Scale parameter for model fitting
+    jfactor : `~astropy.units.Quantity`
+        Integrated J-Factor needed when `~gammapy.modeling.models.PointSpatialModel`
+        spatial model is used
+    z: float
+        Redshift value
+
+    Examples
+    --------
+    This is how to instantiate a `DarkMatterAnnihilationSpectralModel` model::
+
+        from astropy import units as u
+        from gammapy.astro.darkmatter import DarkMatterDecaySpectralModel
+
+        channel = "b"
+        massDM = 5000*u.Unit("GeV")
+        jfactor = 3.41e19 * u.Unit("GeV cm-2")
+        modelDM = DarkMatterDecaySpectralModel(mass=massDM, channel=channel, jfactor=jfactor)  # noqa: E501
+
+    References
+    ----------
+    * `2011JCAP...03..051 <https://ui.adsabs.harvard.edu/abs/2011JCAP...03..051C>`_
+    """
+
+    LIFETIME_AGE_OF_UNIVERSE = 4.3e17 * u.Unit("s")
```

Comment:
This quantity should be be configurable? Is there some science case that could lead to a change of it?

Reply:
Not sure but the decay width is the inverse of the lifetime and that means that when finding an upper limit for the width you can easily have upper limits being much smaller than 1e-17 (normally something like 1e-25). 
This results in the scale parameter being very small during the fitting procedure. Maybe it's numerically more stable to let the user choose a smaller lifetime in respect to having a larger scale parameter